fingerprint/capture/src/main/java/com/simprints/fingerprint/capture/usecase/IsNoFingerDetectedLimitReachedUseCase.kt

```diff
@@ -0,0 +1,26 @@
+package com.simprints.fingerprint.capture.usecase
+
+import com.simprints.fingerprint.capture.state.CaptureState
+import com.simprints.infra.config.store.models.FingerprintConfiguration
+import javax.inject.Inject
+
+internal class IsNoFingerDetectedLimitReachedUseCase @Inject constructor() {
+    operator fun invoke(
+        fingerState: CaptureState,
+        configuration: FingerprintConfiguration
+
+    ): Boolean = when (fingerState) {
+        is CaptureState.ScanProcess -> {
+            val noFingerDetectedThreshold =
+                configuration.secugenSimMatcher?.maxCaptureAttempts?.noFingerDetected?.takeIf { it > 1 }
```

Comment:
Is this feature only applicable to SimMatcher? If so, we should take care to either not call this use case when using NEC or pass the SDK used currently and skip the check for NEC.

Reply:
That's a good question, actually. Let's take it to Slack first

Reply:
OK, then the use case should lookup the configuration for the particular SDK used. Currently it's always checking the SimMatcher config.

Reply:
Great spot! Fixing it now